Gopkg.toml

```diff
@@ -88,7 +88,9 @@ required = [
 
 [[constraint]]
   name = "github.com/cloudevents/sdk-go"
-  version = "0.10.0"
+  # HEAD as of 2019-12-13. Needed the master for this fix: https://github.com/cloudevents/sdk-go/pull/272.
+  # Should be switched back to a release, once a new one is available.
+  branch = "master"
```

Comment:
@n3wscott are you planning on creating a new release for the SDK anytime soon?

Reply:
not much has changed

Reply:
Is https://github.com/cloudevents/sdk-go/pull/272 not enough to warrant a point release?